routes.py

```diff
@@ -579,3 +580,39 @@ def getGroupsJSON():
             "username": username,
         }
     )
+
+
+@app.route('/submitReview', methods=['POST'])
+def submit_review():
+    # username = "proxy"
+    username = CASClient().authenticate()
+    valid_ratings = ['0', '1', '2', '3', '4', '5']
+    building_name = request.form['building-name']
+    room_no = request.form['room-number']
+    overall_rating = request.form['overall-rating']
+    written_review = request.form['written-review']
+    first_checkbox = request.form.getlist('submission-check-1')
+    second_checkbox = request.form.getlist('submission-check-2')
+    # restrict user to one review per room
+    user_search = db_session.query(Reviews).filter(Reviews.building_name == building_name,
+                                                   Reviews.room_number == room_no,
+                                                   Reviews.net_id == username).first()
+    if user_search:
+        message = "You can only submit at most one review per room. Contact it.admin@tigerapps.org to edit your current review."
```

Comment:
perhaps we should mention that you can only submit one review per room, **before the student submits their review** (e.g. maybe add this as a clause to their first condition?). this will encourage ppl to think more carefully about their review before submitting.

Reply:
also, i'm a litte uncertain whether we should be encouraging students to contact us to edit their current review -- i can imagine that students might request trivial edits to their reviews (e.g. changing the rating from 2 to 3, or fixing spelling errors), which we don't have time for. 

i can see 2 alternatives
- modify the current wording to emphasize that they should contact us, only if their edits are "significant" and "necessary"
- or, if a student attempts to submit another review, this new review would override their old one. but the student would have to confirm that this is the desired outcome.

thoughts? @benchanjamin @nicholaspad

Reply:
I like the second option. I will implement that asap.

Reply:
i agree with the second option

Reply:
> i agree with the second option, in which case i don't think we have to have a checkbox confirming that only one review per room can be submitted

confirmation box just implemented for those who are submitting a review again for the same room

Reply:
done